amp/WEB-INF/src/java/resources/application.properties

```diff
@@ -348,6 +348,7 @@ error.aim.updateWorkspace.noRelatedTeam = <font color="red">Specify related Team
 error.aim.updateWorkspace.bilatTeamSelected = <font color="red">Select a multilateral related Team</font>
 error.aim.updateWorkspace.multilatTeamSelected = <font color="red">Select a bilateral related Team</font>
 error.aim.membersExistForTeam=<font color="red">Please remove members from the workspace before removing the workspace</font>
+error.aim.membersRemovedExistForTeam=<font color="red">This workspace has members logically deleted.</font>
```

Comment:
change for members disabled its more clear

Reply:
done